myapp.rb

```diff
@@ -89,71 +89,71 @@ def current_user
   erb :timeline, layout: :layout
 end
 
-get '/register' do
+get "/register" do
   erb :register, layout: :layout
 end
 
-post '/register' do
+post "/register" do
   user = User.new(
     username: params[:username],
     email: params[:email],
     password: params[:password],
-    password_confirmation: params[:password2]
+    password_confirmation: params[:password2],
   )
   if user.save
-    flash[:success] = 'You were successfully registered and can login now'
-    redirect('/login')
+    flash[:success] = "You were successfully registered and can login now"
+    redirect("/login")
   else
-    errors = user.errors.map(&:full_message).join(', ')
+    errors = user.errors.map(&:full_message).join(", ")
     flash[:error] = errors
   end
-  redirect('/register')
+  redirect("/register")
 end
 
-get '/login' do
+get "/login" do
   erb :login, layout: :layout
 end
 
-post '/login' do
+post "/login" do
   user = User.find_by_username(params[:username])
   if user.nil?
-    error = 'Invalid username'
+    error = "Invalid username"
   elsif !user.authenticate(params[:password])
-    error = 'Invalid password'
+    error = "Invalid password"
   else
     session[:user_id] = user.id
-    flash[:success] = 'You were logged in'
-    redirect('/')
+    flash[:success] = "You were logged in"
+    redirect("/")
   end
   flash[:error] = error
-  redirect('/login')
+  redirect("/login")
 end
 
-get '/logout' do
+get "/logout" do
   session[:user_id] = nil
 
-  'You were logged out'
+  "You were logged out"
 end
 
-post '/add_message' do
+post "/add_message" do
   if !session[:user_id]
-    return status 401
+    return status(401)
```

Comment:
Is `status` not ignored alongside `body` and `header` in the `.rubocop.yml`? Why are parentheses needed here?